docs/cri/registry.md

```diff
@@ -185,3 +181,10 @@ DEBU[0000] PullImageRequest: &PullImageRequest{Image:&ImageSpec{Image:gcr.io/you
 DEBU[0001] PullImageResponse: &PullImageResponse{ImageRef:sha256:78096d0a54788961ca68393e5f8038704b97d8af374249dc5c8faec1b8045e42,}
 Image is up to date for sha256:78096d0a54788961ca68393e5f8038704b97d8af374249dc5c8faec1b8045e42
 ```
+
+---
+
+NOTE: The configuration syntax used in this doc is in version 2 which is the
```

Comment:
It would be ok to have a link to an archived v1 description for pre 1.3 users.  We don't support moving a v2 config to the old syntax.